src/content/it/fundamentals/performance/rendering/index.md

```diff
@@ -0,0 +1,151 @@
+di frame scende e il contenuto sono schiaccaiti sullo schermo. Questo è
+spesso indicato come **jank** e influisce negativamente sull'esperienza
+dell'utente.
+
+## La pixel pipeline
+
+Ci sono cinque aree principali che è necessario conoscere e tenere a
+mente a quando lavori. Sono aree sulle quali hai più controllo ed i
+punti chiave nella pipeline pixel-to-screen:
+
+<img src="images/intro/frame-full.jpg"  alt="Tutta la pixel pipeline">
+
+* **JavaScript**. Tipicamente JavaScript è utilizzato per gestire il
+lavoro che determinerà le modifiche visive, che si tratti di
+funzioni jQuery `animate`, orinare set di dati o aggiungere elementi DOM
+alla pagina. Non dovrebbe essere JavaScript ad innescare una modifica
+visiva tuttavia CSS Animations, Transitions e Web Animations API sono
+comunemente utilizzate.
+* **Calcoli Style**. Questo processo determina quali regole CSS si
+applicano a quali elementi basati su selettori di corrispondenza, ad
+esempio, `.headline` o `.nav> .nav__item`. Dopodiché, una volta note le
+regole, vengono applicate e vengono calcolati gli stili finali per ogni
+elemento.
+* **Layout**. Una volta che il browser conosce quali regole si applicano
+ad un elemento può cominciare a calcolare quanto spazio occupa e dove
+sullo schermo. Il modello di layout web significa che un elemento può
+influenzare gli altri, ad esempio la larghezza dell'elemento `body` in
+genere colpisce le larghezze dei suoi figli e così via fino alla fine
+dell'albero, in modo che il processo possa essere abbastanza complesso
+per il browser.
+* **Paint**. Il Painting è il processo di riempimento di pixel. Esso
+implica il disegno di testo, colori, immagini, bordi e ombre,
+sostanzialmente ogni parte visiva degli elementi. Il disegno è
+tipicamente eseguito su superfici multiple, spesso denominate layer.
+* **Compositing**. Poiché le parti della pagina sono state disegnate in
+livelli potenzialmente multipli, devono essere disegnate sullo schermo
+nell'ordine corretto in modo che la pagina sia corretta. Ciò è
+particolarmente importante per gli elementi che si sovrappongono tra
+loro, poiché un errore potrebbe causare un elemento sovrapposto in
+maniera errata ad un altro.
+
+Ognuna di queste parti della pipeline rappresenta un'occasione per
+introdurre jank, quindi è importante capire esattamente quali parti
+della pipeline il codice innesca.
+
+A volte puoi sentire utilizzare il termine "rasterize" in
+combinazione con il paint. Questo perché la pittura è in realtà due
+compiti: 1) creare un elenco delle chiamate al disegno e 2) riempimento
+dei pixel.
+
+Quest'ultimo è chiamato "rasterization" e così ogni volta che vedi paint
+record in DevTools, dovresti pensarli come inclusivi di rasterizzazione.
+(In qualche architettura la creazione delle chiamate al disegno e
+rasterizing vengono eseguite diversi thread, ma non è qualcosa sotto il
+controllo degli sviluppatori.)
+
+Non dovrete sempre toccare ogni parte della pipeline per ogni frame.
```

Comment:
Non dovrai necessariamente essere coinvolto in ogni parte della pipeline per ogni frame.